cardano-cli/src/Cardano/CLI/Shelley/Run/Key.hs

```diff
@@ -244,8 +244,9 @@ runConvertToNonExtendedKey evkf (VerificationKeyFile vkf) =
   writeToDisk
```

Comment:
I think this function could accept also error type constructor and could be used everywhere when the:
```haskell
firstExceptT e . newExceptT $ writeLazyByteStringFile vkf' $ textEnvelopeToJSON desc vk
```
is used now.

Reply:
I prefer not to mingle functional code with error handling if possible.

In some of the changes I've kept the `firstExceptT e . newExceptT` rather than grouping them also to minimise changes.

We can group them to simplify the code.